docs/reference/glossary-of-terms.md

```diff
@@ -0,0 +1,215 @@
+# Glossary of Terms
+
+This document is a list of terms that you might encounter in the ontology world. It is not an exhaustive list and will continue grow and evolve. Please do create a ticket if there is a term you find missing here, or a term that you encounter that you do not understand, and we will do our best to add them in here. This list is not arranged in any particular order, please use the search function to find needed term.
+
+Acknowledgement: Many terms are taken directly from [OAK documentation](https://incatools.github.io/ontology-access-kit/glossary.html) with the permission of Chris Mungall. Many descriptions are also taken from https://www.w3.org/TR/owl2-syntax/
+
+#### Ontology
+A flexible concept loosely encompassing any collection of [Ontology elements](#Ontology-element) and statements or relationships connecting them.
+
+#### Knowledge Graph
+A network of real-world entities—i.e. objects, events, situations, or concepts—and illustrates the relationship between them. Knowledge graphs (in the ontologies world) are thought of as real data built using an ontology as a framework.
```

Comment:
Minor edits:
A network of real-world entities (i.e., objects, events, situations, and concepts) that illustrates the relationships between them.

Reply:
> Knowledge graphs (in the ontologies world) are thought of as real data built using an ontology as a framework.

The second use of the term "ontology world"... what is an "ontology world"? Are there knowledge graphs that have a different meaning that exist outside of this "world"?

Reply:
I made this revision:
A network of real-world entities (i.e., objects, events, situations, and concepts) that illustrates the relationships between them.